reference/commands/config.rst

```diff
@@ -35,3 +39,65 @@ Manages conan.conf information.
     $> 10
 
 
+.. _conan_config_install:
+
+
+conan config install
+____________________
+
+The ``config install`` is intended to share the Conan client configuration. For example, in a company or organization,
+is important to have common ``settings.yml``, ``profiles``, etc.
+
+It retrieves a zip file from a local directory or url and apply the files in the local Conan configuration.
+
+The zip can contain only a subset of all the allowed configuration files, only the present files will be
+replaced, except the **conan.conf** file, that will apply only the declared variables in the zipped ``conan.conf`` file
+and will keep the rest of the local variables.
+
+The **profiles files**, that will be overwritten if already present, but won't delete any other profile file that the user
+has in the local machine.
+
+
+Allowed files in the zipfile:
+
++--------------------------------+----------------------------------------------------------------------+
+| File                           |                                                                      |
++================================+======================================================================+
+| pylintrc                       | Overrides the local ~/.conan/pylintrc                                |
```

Comment:
Local pylintrc isn't a thing. It is just another location to be pointed by a conf variable if desired, but it is not a default, nor it is loaded automatically.

Reply:
fixed